core/src/epicli/data/common/ansible/playbooks/roles/vault/files/configure-vault.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Description: This script perform configuration of Hashicorp Vault to be used with Epiphany
+# More information you can find in Epiphany documentation in HOWTO.md
+# TODO: Revoke root token
+# TODO: Policy for non-root access
+
+HELP_MESSAGE="Usage: configure-vault.sh -c SCRIPT_CONFIGURATION_FILE_PATH -a VAULT_IP_ADDRESS"
+
+function print_help { echo "$HELP_MESSAGE"; }
+
+function log_and_print {
+    local string_to_log="$1";
+    echo "$string_to_log" | tee -a configure-vault.log;
+}
+
+function exit_with_error {
+    local string_to_log="$1";
+    log_and_print "$string_to_log";
+    exit 1;
+}
+
+function check_vault_error {
+    local exit_code ="$1";
+    local success_message="$2";
+    local failure_message="$3";
+    if [ "$exit_code" != "1" ] ; then
+        log_and_print "$success_message";
+    else
+        exit_with_error "$failure_message";
+    fi
+}
+
+function unseal_vault {
+  local init_file_path="$1";
+  log_and_print "Unsealing Vault.";
+  grep -m 3 Unseal "$init_file_path" | awk '{print $4}' | while read -r line ; do
+    vault operator unseal "$line";
+    check_vault_error "$?" "Unseal performed." "There was an error during unsealing of Vault.";
+  done
+}
+
+function enable_vault_audit_logs {
+    log_and_print "Enabling auditing.";
+    vault audit list | grep "file";
+    local COMMAND_RESULT=(${PIPESTATUS[@]})
+    if [ "$COMMAND_RESULT[0]" = "1"] ; then
+        exit_with_error "There was an error during listing auditing.";
+    fi
+    if [ "$COMMAND_RESULT[1]" = "1" ] ; then
+        vault audit enable file file_path="/var/log/vault_audit.log";
+        check_vault_error "$?" "Auditing enabled." "There was an error during enabling auditing.";
+    fi
+}
+
+function mount_secret_path {
+    local secret_path="$1";
```

Comment:
Inconsistent variable naming, please consider converting it to upper case letters.

Reply:
Retracted due to "naming convetions".